src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,33 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
-    public int getUserScore(String[] records, String email) {
-        return 0;
+    public int getUserScore(String[] records, String email) throws UserNotFoundException {
+        if (!isExit(records, email)) {
+            throw new UserNotFoundException("User with given email doesn't exist");
+        }
+        return getRecords(records, email);
+    }
+
+    private boolean isExit(String[] records, String email) {
```

Comment:
we can simplify this solution) Do we need methods isExit() and getRecords()?
Let's move all logic to getUserScore() method